src/core/SampleBuffer.cpp

```diff
@@ -1017,30 +1017,63 @@ void SampleBuffer::visualize(
 	const float ySpace = h * 0.5f;
 	const int nbFrames = focusOnRange ? toFrame - fromFrame : m_frames;
 
-	const int fpp = qBound<int>(1, nbFrames / w, 20);
-	QPointF * l = new QPointF[nbFrames / fpp + 1];
-	QPointF * r = new QPointF[nbFrames / fpp + 1];
+	const int fpp = qBound<int>(1, nbFrames / w, 1000);
```

Comment:
Why limit the frames per pixel to 1000 frames?

Reply:
I tried removing the limit entirely, but for some reason I get a SEGFAULT when I do that.

Reply:
I also tried a value of 10,000, which worked until I zoomed out.  There's something that's causing extremely large values of `fpp` to crash.

Reply:
Oh, that was a bug.  Sometimes `frame + i` wound up being larger than `last`, forgot to check for that.  I can remove that limit now.